xclim/indicators/land/_streamflow.py

```diff
@@ -12,44 +8,33 @@
 __all__ = ["base_flow_index", "freq_analysis", "stats", "fit", "doy_qmax", "doy_qmin"]
 
 
-class Streamflow(Indicator):
+class Streamflow(Daily):
```

Comment:
It can be a good idea to add the indicators inheritance changes to the changelog, if any regression later on. Also, any regression anticipated, specifically on this change? (applies to other places)

Reply:
Done.